tests/cpp/common/test_gpu_hist_util.cu

```diff
@@ -1,50 +1,62 @@
-#include "../../../src/common/device_helpers.cuh"
-#include "../../../src/common/hist_util.h"
-#include "gtest/gtest.h"
-#include "xgboost/c_api.h"
 #include <algorithm>
 #include <cmath>
+
+#include "gtest/gtest.h"
+
 #include <thrust/device_vector.h>
 #include <thrust/iterator/counting_iterator.h>
 
+#include "xgboost/c_api.h"
+
+#include "../../../src/common/device_helpers.cuh"
+#include "../../../src/common/hist_util.h"
+
+#include "../helpers.h"
+
 namespace xgboost {
 namespace common {
 
-void TestDeviceSketch(const GPUSet& devices) {
+void TestDeviceSketch(const GPUSet& devices, bool use_external_memory = false) {
```

Comment:
Do you need a default parameter value here? Wouldn't updating calls to `TestDeviceSketch` be easier?

Reply:
i do not see an issue either way. i'll change it nonetheless.